src/actions/endpointExplorer.js

```diff
@@ -27,11 +27,21 @@ export function updateValues(param, values) {
 
 export const START_REQUEST = "START_REQUEST"
 export const FINISH_REQUEST = "FINISH_REQUEST"
-export function submitRequest(request) {
+export function submitRequest(url, method, formData) {
```

Comment:
I don't like this API change.  In other areas of the application, the term `request` refers to a javascript object (for example, the `buildRequest` method in `EndpointExplorer`) and is handled passed between units as a single object (for example as a param to `EndpointSetup`), whereas here we're saying a request is a triplet of `(url,method,formData)`.

You should pick one or the other.  Either the components should take 3 params instead of one, or submitRequest should take an object named `request`.